modules/images-allow-pods-to-reference-images-from-secure-registries.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+// * openshift_images/using-image-pull-secrets
+
+[id="images-allow-pods-to-reference-images-from-secure-registries_{context}"]
+= Allowing Pods to reference images from other secured registries
+
+The  `.dockercfg` file `$HOME/.docker/config.json` for newer Docker
```

Comment:
```suggestion
The  `.dockercfg` file `$HOME/.docker/config.json` for Docker
```

@rhatdan @TomSweeneyRedHat does `podman login` or `buildah login` write to this file?

Reply:
No.  Podman/Buildah login writes/reads to `${XDG_RUNTIME_DIR}/containers/auth.json`.  When checking credentials Buildah and Podman first check through this file, then if a valid authentication is not found there, they will then check `$HOME/.docker/config.json`.  But neither writes to that `$HOME/.docker/config.json` file, only `docker login` does.

Reply:
@adambkaplan do you think this module just needs to be rewritten?